backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/types/Administrator.kt

```diff
@@ -0,0 +1,6 @@
+package app.ehrenamtskarte.backend.auth.webservice.schema.types
+
+class Administrator(
+    val id: Int,
```

Comment:
I'm not sure, whether we should leak the internal id of the user. Shouldn't matter to the outside world I guess...

Reply:
I would like to keep it. Does not matter to the outside world, but to the inside. If for example we allow to change the email address at one point, the JWT still stays valid (and should), but we would fail to look up the user in our database. It makes everything easier (and probably faster) for us if we use the ID and not the email to identify users.

And I see no risk at publishing the ID. If we really would care about not giving away any metadata at all, we could use UUIDs.